Index.html

```diff
@@ -68,6 +68,243 @@ <h2>I’m Hussen Glad to see you!</h2>
           </ul>
         </div>
       </section>
+      <section class="work-container">
+        <!-- First card -->
+        <div class="card-work">
+          <div class="card-prtfolio">
+            <img src="img/work-img/Snapshoot Portfolio.png" />
+          </div>
+          <div class="work-block">
+            <div class="left-blok">
+              <h2 class="project-title">Tonic</h2>
+              <ul class="frames">
+                <li class="frames-list">
+                   <h2>CANOPY</h2>
+                  </li>
+                  <li class="frames-list">
+                    <img src="img/work-img/Counter.png" />
+                   </li>
+                   <li class="frames-list">
+                    <h2>Back End Dev</h2>
+                   </li>
+                   <li class="frames-list">
+                    <img src="img/work-img/Counter.png" />
+                   </li>
+                   <li class="frames-list">
+                    <h2>2015</h2>
+                   </li>
+               </ul>
+               <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               
+               <ul class="tag-block">
+                <li class="tag-list">
+                   <h2>html</h2>
+                  </li>
+                   <li class="tag-list">
+                    <h2>css</h2>
+                   </li>
+                   <li class="tag-list">
+                    <h2>javaScript</h2>
+                   </li>
+               </ul>
+                <div class="button-acction">
+                    <button class="button-acction-button" type="button"> See Project</button>
+               </div>
+            </div>
+            </div>
+            </div>
+          </div>
+        </div>
+        <!-- Second card  -->
+        <div class="card-work">
```

Comment:
- [ ] I kindly suggest that you follow the Figma design closely by implementing these changes:
- Kindly use the same font of Poppins as seen in the Figma design. Kindly find [Poppins](https://fonts.google.com/share?selection.family=Inter:wght@200;300;400;500;600;700) here from Google. 
- Kindly import Poppins to the top of your css stylesheet file like this: 
`<style>
  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700&display=swap');
</style>`
- Kindly use the same font color for `Back End Dev` and `2015` texts
- Kindly make sure your list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are the same in design as the Figma's
- Kindly make sure your `See Project` button is the same design as the Figma.


| Your Template  | Design Template |
| ------------- | ------------- |
| <img width="246" alt="image" src="https://user-images.githubusercontent.com/52934128/221383281-7f1b7c5c-c747-4139-820c-b5b95a4a15bd.png"> | <img width="337" alt="image" src="https://user-images.githubusercontent.com/52934128/221383354-9afbcc4c-6074-4161-a203-6456c01ff00e.png">  |